main/zoe/guide/contracts/barter-exchange.md

```diff
@@ -1,17 +1,144 @@
 <Zoe-Version/>
 
-##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/958a2c0a3dec38bdba2234934119ea2c28958262/packages/zoe/src/contracts/barter-exchange.js) (Last updated: 6/30/2020)
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/barterExchange.js (Last updated: 6/30/2020)
```

Comment:
So in the atomicSwap doc, it was a link to all contracts. I like linking to the individual contract, but it seems like we need to be consistent about that